packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -296,12 +296,16 @@ export default class ScriptTransformer {
     }
 
     if (!transformed.map) {
-      //Could be a potential freeze here.
-      //See: https://github.com/facebook/jest/pull/5177#discussion_r158883570
-      const inlineSourceMap = sourcemapFromSource(transformed.code);
+      try {
+        //Could be a potential freeze here.
+        //See: https://github.com/facebook/jest/pull/5177#discussion_r158883570
+        const inlineSourceMap = sourcemapFromSource(transformed.code);
 
-      if (inlineSourceMap) {
-        transformed.map = inlineSourceMap.toJSON();
+        if (inlineSourceMap) {
+          transformed.map = inlineSourceMap.toJSON();
+        }
+      } catch (e) {
+        // Error processing the source map; proceed as if it doesn't exist.
```

Comment:
We should at least warn in this case, no? likely a bug in a transformer. Will also mess up breakpoints and code coverage

Reply:
Yes. I've added a warning via console.warn (not sure if that's the preferred method, or if we should add chalking, etc.).